### PR TITLE
LLVM and SPIRV-LLVM-Translator pulldown (WW14 2024)

### PR DESCRIPTION
**Pull request by iclsrc**
LLVM: llvm/llvm-project@417a068b63c0
SPIRV-LLVM-Translator: KhronosGroup/SPIRV-LLVM-Translator@6e62da7a514be99

### CODE REVIEW

**Comment by jsji** [comment]
@bader @intel/llvm-gatekeepers  This is ready for merge. No require required. Thanks.

**Comment by bader** [comment]
@tfzhu, could you check why `/merge` command haven't worked, please?

File: libclc/CMakeLists.txt [hunk 1]
```diff
@@ -174,20 +174,19 @@ FindToolBinary( "${BINARY_VARIABLES}" "${BINARY_NAMES}"
...
-message( "" )
-if( NOT LLVM_CLANG OR NOT LLVM_OPT OR NOT LLVM_AS OR NOT LLVM_LINK
-    OR NOT LIBCLC_REMANGLER )
-  message( FATAL_ERROR "toolchain incomplete!" )
+message( STATUS "libclc toolchain - clang: ${LLVM_CLANG}" )
+message( STATUS "libclc toolchain - llvm-as: ${LLVM_AS}" )
+message( STATUS "libclc toolchain - llvm-link: ${LLVM_LINK}" )
+message( STATUS "libclc toolchain - opt: ${LLVM_OPT}" )
+message( STATUS "libclc toolchain - llvm-spirv: ${LLVM_SPIRV}" )
+if( NOT LLVM_CLANG OR NOT LLVM_OPT OR NOT LLVM_AS OR NOT LLVM_LINK )
```
[the remark this answers is not in the document]
Comment (jsji) [reply libclc/CMakeLists.txt]: Thanks @frasercrmck ! Done.

**Review by bb-sycl — APPROVED** [review]

**Comment by bb-sycl** [comment]
Sat 06 Apr 2024 02:19:42 PM UTC --- Start to merge the commit into sycl branch. It will take several minutes.

**Comment by bb-sycl** [comment]
Sat 06 Apr 2024 02:24:09 PM UTC --- Merge the branch in this PR to base automatically. Will close the PR later.

**Pull request merged**